algorithms task (8)/student/tests/unit_tests/src/maintest.cpp

```diff
@@ -77,3 +77,21 @@ TEST_P(SkillsMatrix, MatrixTopGroups)
     EXPECT_TRUE(std::any_of(topGroups.begin(), topGroups.end(),
                             equalsExpected));
 }
+
+TEST_F(SkillsMatrix, GroupWeights)
+{
+    const std::unordered_map<CVGroup, size_t> expectedGroupWeights = {
+        { CVGroup::CPP, 11 },
+        { CVGroup::Java, 2 },
+        { CVGroup::GeneralDev, 0 },
+    };
+
+    const std::string cvText = "For example, row CPP: "
```

Comment:
thanks

Reply:
And thanks for all your efforts to review all these merge requests! It was very useful for me!